controller/volume_controller.go

```diff
@@ -1456,7 +1456,23 @@ func (c *VolumeController) requestRemountIfFileSystemReadOnly(v *longhorn.Volume
 	if v.Status.State == longhorn.VolumeStateAttached && e.Status.CurrentState == longhorn.InstanceStateRunning {
 		fileSystemReadOnlyCondition := types.GetCondition(e.Status.Conditions, imtypes.EngineConditionFilesystemReadOnly)
 
-		if fileSystemReadOnlyCondition.Status == longhorn.ConditionStatusTrue {
+		isPVMountOptionReadOnly := false
+		kubeStatus := v.Status.KubernetesStatus
+		if kubeStatus.PVName != "" {
+			pv, err := c.ds.GetPersistentVolumeRO(kubeStatus.PVName)
+			if err != nil {
+				log.Warnf("Failed to get PV when checking the mount option of the volume")
```

Comment:
```suggestion
				log.WithError(err).Warn("Failed to get PV when checking the mount option of the volume")
```

Reply:
If err is `NotFound`, we can skip the check. WDYT?

Reply:
Yes the current code return no matter what err it is

Reply:
Fixed, thanks